src/app/pages/dashboard/dashboard.page.ts

```diff
@@ -77,18 +76,11 @@ export class DashboardPage extends BasePageComponent {
       employeeId$: this.store$.select(selectEmployeeId).pipe(map(this.getEmployeeNumberDisplayValue)),
       role$: this.store$.select(selectRole).pipe(map(this.getRoleDisplayValue)),
       isOffline$: this.networkStateProvider.isOffline$,
-      notificationCount$: combineLatest([
-        // each of the individual counts can be added to this array
-        this.store$.pipe(
-          select(getTests),
-          // get all slot ids regarded as incomplete from 'tests' slice of state older than 3 days
-          select(getIncompleteTestsSlotOlderThanADay),
-          withLatestFrom(this.store$.pipe(select(getJournalState))),
-          map(([slotIDs, journal]) => getJournalSlotsBySlotIDs(journal, slotIDs)?.length),
-        ),
-      ]).pipe(
-        // Sum all individual counts to determine overall count
-        map((notificationCounts) => notificationCounts.reduce((a, b) => a + b, 0)),
+      notificationCount$: this.store$.pipe(
+        select(getJournalState),
+        withLatestFrom(this.store$.pipe(select(getTests))),
+        map(([journal, tests]) =>
+          getIncompleteTests(journal, tests, this.dateTimeProvider.now(), this.slotProvider).length),
```

Comment:
Why have we removed combineLatest, I specifically added it so the next requirement to add to the count can be added more seamlessly...

We also need to repoint to the unuploaded selector file selector file i.e.

```
      notificationCount$: combineLatest([
        unsubmittedTestSlotsCount$(this.store$, this.dateTimeProvider, this.slotProvider),
      ]).pipe(
        map(sumFlatArray), /* Sum all individual counts to determine, overall count */
      ),
```

`sumFlatArray` being a helper function we can add inside `src/app/shared/sum-number-array.ts`

```
export const sumFlatArray = (
  arr: number[],
): number => arr.reduce((accumulator, currentValue) => accumulator + currentValue, 0);
```